Articles/Blog/HelloHeatEquation.md

```diff
@@ -0,0 +1,226 @@
+# The Heat Equation is the "Hello World" of Scientific Computing
+
+#### Contributed by [Heather M. Switzer](https://github.com/heatherms27 "Heather Switzer GitHub Profile") and [Elsa Gonsiorowski](https://github.com/gonsie "Elsa Gonsiorowski GitHub Profile")
+
+"Hello World!" is a prototypical program that is universally used by programmers to illustrate the basic syntax of a programming language. At a glance, it can demonstrate a programming language's syntactic structure and can indicate how easy or difficult implementing a basic program may be (see several [Hello World implementations at WikiBooks](https://en.wikibooks.org/wiki/Computer_Programming/Hello_world)). For the scientific software community, a programming language is only the most basic building block of a program. Scientific software must use various programming models or math libraries to achieve high performance for any particular problem. These communities need a version "Hello World!" to compare programming options. The Heat Equation, a well known partial differential equation (PDE), is the perfect formula for this task.
+
+## What is the Heat Equation?
+The Heat Equation, also known as the Diffusion Equation, was developed by Joseph Fourier in 1822. Primarily used in the fields of Mathematics and Physics, the equation shows how heat (or something similar) changes over time in a solid medium. While there are many adaptations for different dimensions and scenerios, new programmers are often first introduced to the 1-dimensional version of the Heat Equation as a tutorial problem. Specifically, the Forward-Time, Central-Space (FTCS) version of the equation is used.
+
+The following description is adapted from [a Trinity University lecture](http://ramanujan.math.trinity.edu/rdaileda/teach/s14/m3357/lectures/lecture_2_25_slides.pdf "One-Dimensional Heat Equation").
+
+Suppose you have a thin metal rod along the x-axis starting at position 0 and ending at position _L_. The goal is to model the temperature distribution throughout the rod at a given time, with the following (ideal) assumptions:
+
+* An initial temperature distribution is given
+* Boundary conditions (established tempuratures at each end of the rod) are known
+* Heat only moves horizontally through the bar
+* There are no external heating/cooling sources
+
+<div style="text-align:center"><img width="75%" src="https://github.com/betterscientificsoftware/images/blob/master/Blog_0719_HeatEqnBar.png" /></div>
+
+To model the temperature distribution of the rod, we can use the 1-dimensional Heat Equation given by the formula:
+
+_&part;u / &part;t = &alpha;&nabla; &sup2;u_
+
+Where:
+* _u_ = Temperature at a position, _x_
+* _t_ = time
+* _&alpha;_  = thermal diffusity of the material
+* _&nabla;_ = The LaPlace operator
+
+### FTCS: Assumptions and Discretization
```

Comment:
I think its best to go from science problem (the rod), to abstract math formulation (the equation) to discretization (the "mesh") and finally to algorithm (the FTCS update procedure). It feels like we're kinda muddling the last two steps a bit and I think it makes some sense to decouple them more here.

Reply:
Will fix up (see earlier comment).